inc/change.class.php

```diff
@@ -540,7 +540,7 @@ static function getAllStatusArray($withmetaforsearch = false) {
 
       $tab = [self::INCOMING      => _x('status', 'New'),
                    self::EVALUATION    => __('Evaluation'),
-                   self::APPROVAL      => __('Approval'),
+                   self::APPROVAL      => CommonITILValidation::getTypeName(1),
```

Comment:
I'm not sure about this one, Approval status for Changes doesn't relate to Validation process.
```suggestion
                   self::APPROVAL      => _n('Approval', 'Approvals', 1),
```